### PR TITLE
Add code to return to the url requested before login 302

### PR DESCRIPTION
**Pull request by erdgeist**
if users are redirected to the login page, add a helper to redirect them to their previous url when successfully logged in.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -94,7 +94,11 @@ def scoped_sign_in_path
     if request.path =~ /\/cfp/
       new_cfp_session_path
     else
-      new_session_path
+      if request.get?
+        new_session_path(:return_to => request.path)
```
Comment (zonque) [inline app/controllers/application_controller.rb]: `return_to: request.path` looks much nicer and is also what the code uses elsewhere.

**Comment by zonque** [comment]
Just curious: Is there any particular reason why you chose not to store this information in the cookie?

**Comment by erdgeist** [comment]
Because we usually do have a bunch of tabs open that belong to the same session and I do have no clue how to properly separate information stored in the cookie. I am afraid all tabs end up on the same view after login.

**Comment by manno** [comment]
Was wondering if it's ok to redirect to a user controlled value, but Rails no longer accepts a hash for redirect_to and the login method is not well suited for URL spoofing attacks.

**Pull request closed without merging**